README.md

```diff
@@ -358,6 +358,12 @@ nylas.drafts
     console.log(`Sent ${message.subject}!`);
   });
 
+// Deleting a draft with the draft object
+nylas.drafts.delete(draft);
+
+// Deleting a draft with the draft id and version
+nylas.drafts.delete('123draftid456', {version: 0});
```

Comment:
If `delete` is called from a `drafts` namespace what's stopping us from overriding it? Or is this just one option and it's possible to call `delete` more generically?